runtime/src/lib.rs

```diff
@@ -211,7 +211,7 @@ parameter_types! {
 
     pub BlockLength: frame_system::limits::BlockLength = frame_system::limits::BlockLength
         ::max_with_normal_ratio(5 * 1024 * 1024, NORMAL_DISPATCH_RATIO);
-    pub const SS58Prefix: u8 = 42;
```

Comment:
I believe it would be good to have a short comment saying why this value was chosen, i.e. what prefix it translates to in SS58